docs/en/models/yolov9.md

```diff
@@ -165,21 +181,14 @@ This example provides simple YOLOv9 training and inference examples. For full do
         yolo predict model=yolov9c.yaml source=path/to/bus.jpg
         ```
 
-## Supported Tasks and Modes
-
-The YOLOv9 series offers a range of models, each optimized for high-performance [Object Detection](../tasks/detect.md). These models cater to varying computational needs and accuracy requirements, making them versatile for a wide array of applications.
-
-| Model      | Filenames                                               | Tasks                                        | Inference | Validation | Training | Export |
-| ---------- | ------------------------------------------------------- | -------------------------------------------- | --------- | ---------- | -------- | ------ |
-| YOLOv9     | `yolov9t` `yolov9s` `yolov9m` `yolov9c.pt` `yolov9e.pt` | [Object Detection](../tasks/detect.md)       | ✅        | ✅         | ✅       | ✅     |
-| YOLOv9-seg | `yolov9c-seg.pt` `yolov9e-seg.pt`                       | [Instance Segmentation](../tasks/segment.md) | ✅        | ✅         | ✅       | ✅     |
-
-This table provides a detailed overview of the YOLOv9 model variants, highlighting their capabilities in object detection tasks and their compatibility with various operational modes such as [Inference](../modes/predict.md), [Validation](../modes/val.md), [Training](../modes/train.md), and [Export](../modes/export.md). This comprehensive support ensures that users can fully leverage the capabilities of YOLOv9 models in a broad range of object detection scenarios.
-
 !!! note
 
     Training YOLOv9 models will require _more_ resources **and** take longer than the equivalent sized [YOLOv8 model](yolov8.md).
 
+## Conclusion
```

Comment:
I figured this `Conclusion` is part of the introduction section of yolov9 above, not like a final conclusion for this page, maybe revert this one as well?